sklearn/multiclass.py

```diff
@@ -525,6 +525,14 @@ class OneVsOneClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     estimators_ : list of ``n_classes * (n_classes - 1) / 2`` estimators
         Estimators used for predictions.
 
+    coef_ : ndarray of shape (1, n_features) or (n_classes, n_features)
+        Coefficient of the features in the decision function. Set only when
+        ``estimator`` defines ``coef_``.
```

Comment:
I know I have suggested something else before. I think this is a little more clear:

```suggestion
        Coefficient of the features in the decision function. This attribute exists
        only if the ``estimators_`` defines ``coef_``.
```